docs/source/conf.py

```diff
@@ -70,3 +85,28 @@
 # relative to this directory. They are copied after the builtin static files,
 # so a file named "default.css" will overwrite the builtin "default.css".
 html_static_path = ['_static']
+
+# Breathe + Exhale configuration
+
+# Setup the breathe extension
+breathe_projects = {
+    "FlexFlow": "./_doxygen/xml"
+}
+breathe_default_project = "FlexFlow"
+
+# Setup the exhale extension
+exhale_args = {
+    # These arguments are required
+    "containmentFolder":     "./c++_api",
+    "rootFileName":          "c++_api_root.rst",
+    "doxygenStripFromPath":  "..",
+    # Heavily encouraged optional argument (see docs)
+    #"rootFileTitle":         "Library API",
+    # Suggested optional arguments
+    "createTreeView":        True,
+    # TIP: if using the sphinx-bootstrap-theme, you need
+    # "treeViewIsBootstrap": True,
+    "exhaleExecutesDoxygen": True,
+    #"exhaleDoxygenStdin":    f"INPUT = {os.path.join(FF_HOME, 'include', 'flexflow')}{os.path.join(FF_HOME, 'src')}"
+    "exhaleDoxygenStdin":    f"INPUT = {FF_HOME}/include/flexflow {FF_HOME}/src/loss_functions {FF_HOME}/src/mapper {FF_HOME}/src/metric_functions {FF_HOME}/src/ops {FF_HOME}/src/parallel_ops {FF_HOME}/src/recompile {FF_HOME}/src/utils"
```

Comment:
Why not just `src`? Also could there be any quoting issues here if `FF_HOME` includes a space?

Reply:
What were the issues with `src/runtime`?